content/getting-started/building-an-ionic-app.md

```diff
@@ -0,0 +1,390 @@
+                # CM_CERTIFICATE_PASSWORD: Encrypted(...) # <-- Put your encrypted certificate password here
+                # CM_PROVISIONING_PROFILE: Encrypted(...) # <-- Put your encrypted provisioning profile here
+                #
+                # Automatic Code Signing 
+                # https://docs.codemagic.io/yaml/distribution/
+                # https://appstoreconnect.apple.com/access/api
+                APP_STORE_CONNECT_ISSUER_ID: 5a451239-51eb-10b6-bfcc-60e61ddab13c # <-- Put your App Store Connect Issuer Id here
+                APP_STORE_CONNECT_KEY_IDENTIFIER: LY55E1G322 # <-- Put your App Store Connect Key Identifier here
+                APP_STORE_CONNECT_PRIVATE_KEY: Encrypted(...) # <-- Put your App Store Connect Private Key here
+                CERTIFICATE_PRIVATE_KEY: Encrypted(...) # <-- Put your Certificate Private key here
+            node: latest
+            xcode: latest
+            cocoapods: default
+        triggering:
+            events:
+                - push
+            branch_patterns:
+                - pattern: develop
+                  include: true
+                  source: true
+        scripts:
+            - name: Install npm dependencies for Ionic project
+              script: |
+                npm install
+            - name: Cocoapods installation
+              script: |
+                cd ios/App && pod install
+            - name: Update dependencies and copy web assets to native project
+              script: |
+                # npx cap copy # <- use this is you don't need to update native dependencies
+                npx cap sync # <- update native dependencies and copy web assets to native project
+            - name: Set up keychain to be used for codesigning using Codemagic CLI 'keychain' command
+              script: |
+                keychain initialize
+            # - name: Set up Provisioning profiles from environment variables (Use with manual code signing)
+            #   script: |
+            #     PROFILES_HOME="$HOME/Library/MobileDevice/Provisioning Profiles"
+            #     mkdir -p "$PROFILES_HOME"
+            #     PROFILE_PATH="$(mktemp "$PROFILES_HOME"/$(uuidgen).mobileprovision)"
+            #     echo ${CM_PROVISIONING_PROFILE} | base64 --decode > $PROFILE_PATH
+            #     echo "Saved provisioning profile $PROFILE_PATH"
+            - name: Fetch signing files
+              script: |
+                # app-store-connect fetch-signing-files "com.nevercode.ncionicapp" --type IOS_APP_STORE --create
+                app-store-connect fetch-signing-files $(xcode-project detect-bundle-id) --type IOS_APP_STORE --create
+            # - name: Set up signing certificate (Use with manual code signing)
+            #   script: |
+            #     echo $CM_CERTIFICATE | base64 --decode > /tmp/certificate.p12
+            #     keychain add-certificates --certificate /tmp/certificate.p12 --certificate-password $CM_CERTIFICATE_PASSWORD 
```

Comment:
I know you want to explicitly show two code signing options but having the two options in the file and some lines marked as commentary creates a lot of noise and makes the file difficult to read